docs/tutorials/ApplicationMigration/ApplicationMigration.md

```diff
@@ -469,70 +479,7 @@ spec:
           - op: add
             path: /metadata/annotations
             value:
-              {"service.beta.kubernetes.io/azure-dns-label-name":"fleet-test-member-1"}
-      - clusterSelector:
-          clusterSelectorTerms:
-            - labelSelector:
-                matchLabels:
-                  cluster-name: member-2
-        overrideType: Delete
-```
-
-Summary:
-- The override makes sure the nginx-service won't be applied to the Member Cluster 2 cluster.
-
-#### Create ro-nginx-service-export-2 Override
-
-> Note:  override file located [here,](./testfiles/ro-nginx-service-export-2.yaml) and it should be created before the new service.
-> So that the overrides can be applied to these resources.
-
-To ensure the new service is exposed in Member Cluster 2 only, you need to create an override to only place the serviceExport in Member Cluster 2 when propagating.
-```yaml
-apiVersion: placement.kubernetes-fleet.io/v1alpha1
-kind: ResourceOverride
-metadata:
-  name: ro-nginx-service
-  namespace: test-app
-spec:
-  resourceSelectors:
-    -  group: networking.fleet.azure.com
-       kind: ServiceExport
-       version: v1alpha1
-       name: nginx-service-2
-  policy:
-    overrideRules:
-      - clusterSelector:
-          clusterSelectorTerms:
-            - labelSelector:
-                matchLabels:
-                  cluster-name: member-1
-        overrideType: Delete
-```
-
-#### Create ro-nginx-service-2 Override
-> Note:  override file located [here,](./testfiles/ro-nginx-service-2.yaml) and it should be created before the new service.
-> So that the overrides can be applied to these resources.
-
-```yaml
-apiVersion: placement.kubernetes-fleet.io/v1alpha1
-kind: ResourceOverride
-metadata:
-  name: ro-nginx-service
-  namespace: test-app
-spec:
-  resourceSelectors:
-    -  group: ""
-       kind: Service
-       version: v1
-       name: nginx-service-2
-  policy:
-    overrideRules:
-      - clusterSelector:
-          clusterSelectorTerms:
-            - labelSelector:
-                matchLabels:
-                  cluster-name: member-1
-        overrideType: Delete
+              { "networking.fleet.azure.com/weight": "50" }

```

Comment:
This was used to add the weight when there is nothing. Now that the key already exists, shouldn't it be a replace and only on the value?

Reply:
it will override all the annotations. In this case, it should be fine since export only supports this annotation.